app-android/src/main/res/layout/fragment_school_overview.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <variable
+            name="fieldsEnabled"
+            type="boolean" />
+
+        <variable
+            name="schoolWithHolidayCalendar"
+            type="com.ustadmobile.lib.db.entities.SchoolWithHolidayCalendar" />
+
+        <variable
+            name="loading"
+            type="boolean" />
+
+        <variable
+            name="editButtonMode"
+            type="com.ustadmobile.core.view.EditButtonMode"/>
+
+
+    </data>
+
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/fragment_school_detail_overview_nestedscrollview"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
layout_height here should be match_parent - e.g. the nested scroll view should match the height of the parent, and then the scrolling should be within it (eg as it's child items layout_height is wrap_content)